src/main/java/igrad/logic/commands/requirement/RequirementAddCommand.java

```diff
@@ -31,12 +29,10 @@
 
     public static final String MESSAGE_HELP = MESSAGE_DETAILS + MESSAGE_USAGE;
 
-    public static final String MESSAGE_SUCCESS = "New requirement added: %1$s";
-    public static final String MESSAGE_NOT_ADDED = "Added requirement must be provided with at least these argument(s) "
+    public static final String MESSAGE_REQUIREMENT_ADD_SUCCESS = "New requirement added: %1$s";
```

Comment:
same as above, i'll change the variable naming for modules too, this naming seems more intuitive